lib/job-simulation.rb

```diff
@@ -1,31 +1,58 @@
-require './Stack.rb'
-require './Queue.rb'
+require_relative 'Stack'
+require_relative 'Queue'
 
 class JobSimulation
-  attr_reader :workers, :waiting, :roll
+  attr_reader :employed, :waiting, :waitlist
 
   def initialize (jobs_available, job_seekers)
+    #job_seekers must be larger than jobs_available
+    @waiting = job_seekers - jobs_available
+    give_jobs(jobs_available)
+    create_wait_list
+  end
 
+  def give_jobs(jobs_available)
+    @number = 1
+    @employed = Stack.new
+    jobs_available.times do
+      @employed.push("Worker \##{@number}")
+      @number += 1
+    end
+    @employed
+  end
```

Comment:
It's fine to break out this task into its own method - but if doing so, it should be a private method that is used within the class, but not available outside the class (which is what making it private does)